build-output.eslintrc

```diff
@@ -1,6 +1,6 @@
 {
     "root": true,
-    "plugins": ["promise", "@typescript-eslint"],
+    "plugins": ["promise", "@typescript-eslint", "eslint-plugin-n"],
```

Comment:
I still dunno why this was needed hah, has there been anything changed you can see that correlates?

Reply:
> error  Definition for rule 'n/no-callback-literal' was not found  n/no-callback-literal

was the error from [this run](https://github.com/duckduckgo/content-scope-scripts/actions/runs/5646734760/job/15295293432?pr=648)

Reply:
This test was failing since the first push, I'm not sure what's causing it. I can try to bisect

Reply:
And not locally right? So weird. If you get chance it would be worth a bash.

Reply:
I'm getting this error locally even on the current `main` branch. Maybe some of the external packages changed?

Reply:
Not sure (shouldn't do as the packages should be pinned), or the tests were skipped recently. Let's add for now. Thanks for checking.